docs/source/tutorials/transfer.md

```diff
@@ -0,0 +1,169 @@
+# Tutorial
+
+## Transfer
+
+In this tutorial you will learn how to execute Kagome-based Polkadot-host chain which can be used as a cryptocurrency, and interact with it by sending extrinsics and executing queries.
+
+### Prerequisites
+
+1. Kagome full node binary built as described [here](../overview/getting_started.md#Build full validating node).
+2. For your convenience make sure you have this binary included into your path:
+```
+PATH=$PATH:build/node/kagome_full/
+```
+3. Python 3 installed in the system  
+
+### Tutorial
+
+During this tutorial we will:
+1. Launch Kagome network using prepared genesis file
+2. Query the balance of Alice account
+3. Send extrinsic that transfers some amount of currency from Alice to Bob
+4. Query again the balance of Alice to make sure balance was updated
+
+#### Launch Kagome network
+
+For this tutorial we will spin up a simple network of a single peer with predefined genesis.
+
+To start with let's navigate into the node's folder:
+
+```
+cd node
+```
+
+`node` folder contains necessary configuration files for our tutorial:
+
+1. `config/localchain.json` – genesis file for our network. It contains necessary key-value pairs that should be inserted before the genesis block.
+2. `config/localkeystore.json` – file containing the keys for Kagome peer. This is necessary to be able to sign the messages sent by our validating node. 
+
+localchain.json contains Alice and Bob accounts. Both have 1000000000000000000000 amount of crypto.
+Their keys can be generated using [subkey](https://substrate.dev/docs/en/ecosystem/subkey) tool:
+```
+subkey inspect //Alice
+# Secret Key URI `//Alice` is account:
+# Secret seed:      0xe5be9a5092b81bca64be81d212e7f2f9eba183bb7a90954f7b76361f6edb5c0a
+# Public key (hex): 0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d
+# Account ID:       0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d
+# SS58 Address:     5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
+
+subkey inspect //Bob  
+# Secret Key URI `//Bob` is account:
+# Secret seed:      0x398f0c28f98885e046333d4a41c19cee4c37368a9832c6502f6cfd182e2aef89
+# Public key (hex): 0x8eaf04151687736326c9fea17e25fc5287613693c912909cb226aa4794f26a48
+# Account ID:       0x8eaf04151687736326c9fea17e25fc5287613693c912909cb226aa4794f26a48
+# SS58 Address:     5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty
+```
+
+
+
+---
+**Note**
+
+If you are running this tutorial not for the first time, then make sure you cleaned up your storage as follows:
+```
+rm -rf ldb
+```
+--- 
+
+To execute the chain run run `kagome_full` using the following flags: 
+```bash
+Kagome application allowed options:
```

Comment:
Should we really put this into documentation? A user can always view it with -h, and if the parameters list changes than the docs will become inconsistent.